### PR TITLE
Update waas-overview.md

### PR DESCRIPTION
**Pull request by lindspea**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/4148
Added additional link.

### CODE REVIEW

**Comment by jvsam** [comment]
Hi @nenonix and @JohanFreelancer9 for copy review please. Thank you.

File: windows/deployment/update/waas-overview.md [hunk 1]
```diff
@@ -145,7 +145,7 @@ Specialized systems—such as PCs that control medical equipment, point-of-sale
...
 
 >[!NOTE]
->Windows 10 LTSB will support the currently released processors and chipsets at the time of release of the LTSB. As future CPU generations are released, support will be created through future Windows 10 LTSB releases that customers can deploy for those systems. For more information, see **Supporting the latest processor and chipsets on Windows** in [Lifecycle support policy FAQ - Windows Products](https://support.microsoft.com/help/18581/lifecycle-support-policy-faq-windows-products).
+>Windows 10 LTSB will support the currently released processors and chipsets at the time of release of the LTSB. As future CPU generations are released, support will be created through future Windows 10 LTSB releases that customers can deploy for those systems. For more information, see **Supporting the latest processor and chipsets on Windows** in [Lifecycle support policy FAQ - Windows Products](https://support.microsoft.com/help/18581/lifecycle-support-policy-faq-windows-products) and [Windows Processor Requirements.](https://docs.microsoft.com/windows-hardware/design/minimum/windows-processor-requirements)
```
Comment (illfated) [inline windows/deployment/update/waas-overview.md]: Just a minor nitpick: Move the dot to the end of the sentence (not part of the link).
```suggestion
>Windows 10 LTSB will support the currently released processors and chipsets at the time of release of the LTSB. As future CPU generations are released, support will be created through future Windows 10 LTSB releases that customers can deploy for those systems. For more information, see **Supporting the latest processor and chipsets on Windows** in [Lifecycle support policy FAQ - Windows Products](https://support.microsoft.com/help/18581/lifecycle-support-policy-faq-windows-products) and [Windows Processor Requirements](https://docs.microsoft.com/windows-hardware/design/minimum/windows-processor-requirements).
```

File: windows/deployment/update/waas-overview.md [hunk 2]
```diff
@@ -145,7 +145,7 @@ Specialized systems—such as PCs that control medical equipment, point-of-sale
...
 
 >[!NOTE]
->Windows 10 LTSB will support the currently released processors and chipsets at the time of release of the LTSB. As future CPU generations are released, support will be created through future Windows 10 LTSB releases that customers can deploy for those systems. For more information, see **Supporting the latest processor and chipsets on Windows** in [Lifecycle support policy FAQ - Windows Products](https://support.microsoft.com/help/18581/lifecycle-support-policy-faq-windows-products).
+>Windows 10 LTSB will support the currently released processors and chipsets at the time of release of the LTSB. As future CPU generations are released, support will be created through future Windows 10 LTSB releases that customers can deploy for those systems. For more information, see **Supporting the latest processor and chipsets on Windows** in [Lifecycle support policy FAQ - Windows Products](https://support.microsoft.com/help/18581/lifecycle-support-policy-faq-windows-products) and [Windows Processor Requirements](https://docs.microsoft.com/windows-hardware/design/minimum/windows-processor-requirements).
```
Comment (nenonix) [inline windows/deployment/update/waas-overview.md]:
```suggestion
>Windows 10 LTSB will support the currently released processors and chipsets at the time of release of the LTSB. As future CPU generations are released, support will be created through future Windows 10 LTSB releases that customers can deploy for those systems. For more information about Windows support for the latest processor and chipsets, see  [Windows Processor Requirements](https://docs.microsoft.com/windows-hardware/design/minimum/windows-processor-requirements).
```

**Comment by jvsam** [comment]
Hi @greg-lindsay this pull request has been copy edited and is ready for your final review and approval. Please let us know if there are outstanding issues to be resolved prior to merge. Thank you. See: [commits](https://github.com/MicrosoftDocs/windows-itpro-docs/pull/4343/commits) and [changes](https://github.com/MicrosoftDocs/windows-itpro-docs/pull/4343/files).

**Comment by jvsam** [comment]
Hi again @greg-lindsay, kindly review this PR when time permits. We appreciate the help, thank you.

**Comment by illfated** [comment]
Rich text diff: https://github.com/MicrosoftDocs/windows-itpro-docs/pull/4343/files?short_path=9875375#diff-9875375e9c46c525b8717cef1eeabad5
Ordinary diff: https://github.com/MicrosoftDocs/windows-itpro-docs/pull/4343/files

**Comment by jvsam** [comment]
Hi again @greg-lindsay, hope all is well. Can you please check this PR and if there are no issues with the changes, kindly merge? @AndreaBarr will you be able to assist with the review? Thank you so much.

**Comment by jvsam** [comment]
Hi @AndreaBarr can you please help us with this? This PR contains a more targeted replacement link. Thank you.

File: windows/deployment/update/waas-overview.md [hunk 3]
```diff
@@ -145,7 +145,7 @@ Specialized systems—such as PCs that control medical equipment, point-of-sale
 Microsoft never publishes feature updates through Windows Update on devices that run Windows 10 Enterprise LTSB. Instead, it typically offers new LTSC releases every 2–3 years, and organizations can choose to install them as in-place upgrades or even skip releases over a 10-year life cycle.
 
 >[!NOTE]
```
Comment (illfated) [inline windows/deployment/update/waas-overview.md]: I have read some of Nick Schonning's pull requests recently, especially the MarkDown rule compliance chore PRs, where at least one of them contained a Note format update: always keep one space character between the quote marker (angle bracket `>` ) and its subsequent text.
```suggestion
> [!NOTE]
```
(I also recommend adding the same space in the next line below.)

**Comment by jvsam** [comment]
Hi again @greg-lindsay and @AndreaBarr, I'd like to follow up if this PR can be merged. The suggested update will replace the *Lifecycle FAQ* link with a direct link to the *Windows Processor Requirements* content page. This way, a user wouldn't have to  navigate through the page to find the correct link (the destination page also contains Lifecycle FAQ link). Thank you again.

**Comment by jvsam** [comment]
Thank you very much @AndreaBarr!

**Pull request merged**